### ISSUE TITLE
x/tools: build dashboard is red

### ISSUE THREAD

**Issue by agnivade** [golang/go#26531]
https://build.golang.org/?repo=golang.org%2fx%2ftools.

I see failures across all platforms.

Mostly it is this line which is getting repeated in all the test logs- 

```
 gcimporter_test.go:93: testPath(image): reading export data: /path/to/some_file.a: cannot import, possibly version skew (unknown export format version -1 ("<hex_escaped_text>")) - reinstall package
```

A couple of the failures - 
https://build.golang.org/log/af9e4d76623dc0222c893aea9fb08594d936c271
https://build.golang.org/log/572376bd9dd8e6c81e37c95d587d030c25291b66

Looks like a valid failure which needs to be fixed. Please feel free to correct me if I am wrong.

**Comment by mvdan** [comment]
Perhaps related to this earlier error?

```
# golang.org/x/tools/godoc
/var/folders/dx/k53rs1s93538b4x20g46cj_w0000gn/T/workdir/gopath/src/golang.org/x/tools/godoc/template.go:93: missing ... in args forwarded to printf-like function
```

**Comment by agnivade** [comment]
Perhaps, although that was changed on 13th June. https://github.com/golang/tools/commit/02fcd6aaf1349bf6a3f24ef1b62ca07ff618ff59. I am unable to see all results from the `release-branch.go.1.10` in the build dashboard UI, which might shed more light.

**Comment by bcmills** [comment]
@ianthehat, is this related to the work on x/tools/go/packages?

**Comment by ianthehat** [comment]
I don't believe so, x/tools/go/packages is a higher level abstraction that uses golang.org/x/tools/go/gcexportdata to load export data itself.
The version skew message implies that the compiler and the gcimporter disagreed about the format of the export data, I know it changed in 1.11 but I thought all the code to do with that had been stable for a while now, and was selected by build tag.

**Comment by agnivade** [comment]
I think I have figured it out. It is this test which is actually failing in 1.10 and 1.9.
```
TestWeb
FAIL	golang.org/x/tools/cmd/godoc	10.326s
```

And that is due to @bradfitz's latest changes to show version info in godoc. It is checking for text which has only been added in 1.11. Hence those are failing for 1.10 and 1.9. I am going to repurpose this bug to fix this since I see a clear reason for the failure.

I don't see the gcimporter_test.go failures any more. It is just TestWeb failures now. We can create a new issue if we see that again.
